README.md

```diff
@@ -6,20 +6,42 @@
 
 Source code for https://crystal-ann.com.
 
-## Develop
+## Setup
 
-Create a pg database called `crystal_ann`. Then:
+1. [Install Crystal](https://crystal-lang.org/docs/installation/index.html)
+2. [Install Amber Framework](https://docs.amberframework.org/getting-started/Installation/)
+3. [Install Postgres](http://postgresguide.com/setup/install.html)
+4. Create `crystal_ann` and `crystal_ann_test` pg databases
+
+## Development
+
+1. Install project dependencies:
+
+```
+$ crystal deps
+```
+
+2. Run database migrations:
 
 ```
-$ shards update
 $ amber migrate up
+```
+
+3. Create seed data:
+
+```
 $ crystal db/seed.cr
+```
+
+4. Start app and watch for source changes:
+
+```
 $ amber watch
 ```
 
-## Test
+## Testing
 
-Create a pg database called `crystal_ann_test`. Then:
+Migrate test database and run specs:
```

Comment:
Should this be run with ```AMBER_ENV=test```?

Reply:
No, because of this: https://github.com/crystal-community/crystal-ann/blob/master/spec/spec_helper.cr#L1

I will go again through the setup and double check :)

Reply:
Totally forgot!

It looks good to me :)